package.json

```diff
@@ -71,7 +71,7 @@
     "tslint": "^5.8.0",
     "tslint-microsoft-contrib": "^5.0.0",
     "tsutils": "^2.15.0",
-    "typescript": "^2.6.2"
+    "typescript": "^2.9.1"
```

Comment:
Does not this work with TS@2.8?

Reply:
The snapshot tests won't. This is just a dev dependency.

Reply:
Oh, snapshot... OK.